gene_finder.py

```diff
@@ -45,11 +51,25 @@ def get_reverse_complement(dna):
     >>> get_reverse_complement("CCGCGTTCA")
     'TGAACGCGG'
     """
-    # TODO: implement this
-    pass
-
-
-def rest_of_ORF(dna):
+    # Step 1 get complement
+    counter_1=0
+    complement_dna = ''
+    while counter_1 < len(dna):
+        complement_dna = complement_dna + get_complement(dna[counter_1])
+        counter_1 = counter_1 + 1
+    # Step 2 reverse it
+    reverse_complement = ''
+    counter_2=len(dna)-1
+    while counter_2 >= 0:
+        reverse_complement = reverse_complement + complement_dna[counter_2]
+        counter_2 = counter_2 - 1
+    return reverse_complement
+
+# Define Stop Codons
+stop_codons = ['TAA', 'TAG', 'TGA']
+start_codon = 'ATG'
+
```

Comment:
It's usually recommended to put these defining statements right below the import statements because these variables are going to get referenced in many other functions that come below.